system-contracts/contracts/L2BaseToken.sol

```diff
@@ -128,21 +128,4 @@ contract L2BaseToken is IBaseToken, ISystemContract {
         return abi.encodePacked(IMailbox.finalizeEthWithdrawal.selector, _to, _amount, _sender, _additionalData);
     }
 
-    /// @dev This method has not been stabilized and might be
-    /// removed later on.
-    function name() external pure override returns (string memory) {
```

Comment:
What is the motivation for removing these?

Reply:
you can see in the description of this PR